src/components/product-icon/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { IconBoundaryColor16 } from '@hashicorp/flight-icons/svg-react/boundary-color-16'
 import { IconConsulColor16 } from '@hashicorp/flight-icons/svg-react/consul-color-16'
-import { IconHashicorp16 } from '@hashicorp/flight-icons/svg-react/hashicorp-16'
+import { IconHashicorpColor16 } from '@hashicorp/flight-icons/svg-react/hashicorp-color-16'
```

Comment:
Made this update - icons are equivalent, except the `Color` icon won't change color via `currentcolor`, which is necessary for use in `IconTileLogo`.

Reply:
This change breaks this icon in `ProductSwitcher`:

![image](https://user-images.githubusercontent.com/43934258/151872933-8372c958-c67b-416d-ab83-1a025d641a33.png)

Can you add a special class in there like what's done for Vault to set the color to white in the switcher?

Reply:
For context:
- as Ashlee noticed, the icon for HCP should not be the "color" one (the official "brand" color for this icon is black) but the normal one that can be tinted (in this case white, I imagine).
  - even better, you could use the `hcp-16` icon, that exists exactly for HCP (https://flight-hashicorp.vercel.app/?query=hcp) 
- the vault icon has been recently released (see https://github.com/hashicorp/flight/issues/372) with the correct yellow: https://flight-hashicorp.vercel.app/?query=vault-color

Reply:
Thanks for catching this @ashleemboyer , and thanks for the extra context @didoo ! I've taken the approach Ashlee suggested - we need the "color" icon in place for now to avoid having the color change in `IconTileLogo` contexts.

Have added a `TODO` for us to revisit this in the future - eg perhaps we could expand `ProductIcon` to have a `color={true | false}` option to switch between "color" and tintable icons in the future.